R/on-load.R

```diff
@@ -3,4 +3,9 @@
 
 .onLoad <- function(libname, pathname) {
   debugme()
+
+  shiny::addResourcePath(
+    "shinytest",
+    system.file("js", package = "shinytest")
+  )
```

Comment:
I guess this is only needed in the child process that is running the app, right? So maybe it makes sense to make it conditional on that somehow?